src/index.html

```diff
@@ -6,7 +6,43 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="calendar calendar--start-day-fri">
+    <div class="calendar__month">
+      <div
+        class="calendar__day calendar__day--start-day-sat"
+        title="First day of month"
```

Comment:
not on the first child should start day be specified,
use :first-child for styling this element

```suggestion
        class="calendar__day"
```